dvc/command/data_sync.py

```diff
@@ -20,11 +24,16 @@ def run(self):
                 ret = 1
         return ret
 
+    @staticmethod
+    def check_up_to_date(processed_files_count, message):
+        if processed_files_count == 0:
+            logger.info(message)
```

Comment:
We can safely use `logger.info(self.UP_TO_DATE_MSG)` here, without passing it explicitly. Sorry, looks like I've confused you by mentioning `staticmethod` in my previous review :slightly_frowning_face:

Reply:
Do you mean we need to remove `@staticmethod`? I see `logger` is imported from `dvc.logger`, so `logger` is actually depended on the context.

Reply:
We can leave it and use `logger.info(CmdBase.UP_TO_DATE_MSG)` or use it as a classmethod. The main point is that there is no need to explicitly pass UP_TO_DATE_MSG to check_up_to_date, whenever you call it. I.e.
```
@classmethod
def check_up_to_date(cls, count):
    if count == 0:
        logger.info(cls.UP_TO_DATE_MSG)
```